lib/test_prof/recipes/rspec/let_it_be.rb

```diff
@@ -130,6 +155,9 @@ def let_it_be(identifier, **options, &block)
             super()
           end
         end
+
+        # Used to to track method overrides
+        define_method("let_it_be_#{identifier}") {}
```

Comment:
I don't like this idea. Let's try to use the information we already have to detect if there is already a let_it_be-defined method.

For example, we can check that the `instance_method_defined?(identifier)` and then check the location of the definition, and if it's the current file—it's defined by `let_it_be`. Something like that.

Reply:
Ah, I didn't realize that we can check the source file where the method is defined

Done